semantic-segmentation-deeplabv3/script/score.py

```diff
@@ -0,0 +1,127 @@
+import os
+import numpy as np
+from PIL import Image
+from io import BytesIO
+from argparse import ArgumentParser
+import base64
+import pandas as pd
+import pyarrow.parquet as pq  # noqa: F401
+from azureml.studio.modulehost.handler.port_io_handler import OutputHandler
+from azureml.studio.common.datatypes import DataTypes
+from azureml.studio.common.datatable.data_table import DataTable
+from azureml.studio.common.io.data_frame_utils import load_data_frame_from_directory
+
+import torch
+from torchvision import transforms
+
+from .model import SegmentationNet
+from .smt_fake import smt_fake_file
+
+
+class Score:
+    def __init__(self, model_path, meta={}):
+        self.pallete = [
+            [255, 255, 255],
+            [128, 64, 128],
+            [244, 35, 232],
+            [70, 70, 70],
+            [102, 102, 156],
+            [190, 153, 153],
+            [153, 153, 153],
+            [250, 170, 30],
+            [220, 220, 0],
+            [107, 142, 35],
+            [152, 251, 152],
+            [70, 130, 180],
+            [220, 20, 60],
+            [255, 0, 0],
+            [0, 0, 142],
+            [0, 0, 70],
+            [0, 60, 100],
+            [0, 80, 100],
+            [0, 0, 230],
+            [119, 11, 32],
+            [0, 0, 0]
+        ]
+
+        self.inference_transforms = transforms.Compose([
+            transforms.ToTensor(),
+            transforms.Normalize([.485, .456, .406], [.229, .224, .225]),
+        ])
+        self.model = SegmentationNet(model_type=meta['Model type'], model_path=model_path, pretrained=False)
+        self.model.load_state_dict(torch.load(os.path.join(model_path, 'model.pth'), map_location='cpu'))
+        if meta['Use CUDA'] and torch.cuda.is_available():
+            self.model = self.model.cuda()
+
+        self.model.eval()
+
+    def run(self, input, meta=None):
+        my_list = []
+        for i in range(input.shape[0]):
```

Comment:
Use func like 'decode_image_str' to convert https://github.com/hirofumi-s-friends/CustomModules/blob/chjinche/add-image-classification-densenet/densenet-img-cls/densenetimgcls/score.py

Reply:
Done!